client/data/testRides.tsx

```diff
@@ -0,0 +1,20 @@
+const rides = [
+    {
+        'ride_id': '1234',
+        'start_time': '2023-01-01 00:00:00', // Placeholder: Pretty sure firestore doesn't store timestamps this way
```

Comment:
You might be able to use the firestore Timestamp class: firebase.firestore.Timestamp.now()